docs/src/main/asciidoc/getting-started-reactive.adoc

```diff
@@ -292,6 +292,24 @@ public class ReactiveGreetingResource {
 The `ReactiveGreetingService` class contains a straightforward method producing a `Uni`.
 While, in this example, the resulting item is emitted immediately, you can imagine any async API producing a `Uni`. We cover this later in this guide.
 
+NOTE: In order to get Mutiny working properly with JAX-RS resources make sure the RESTEasy Mutiny extension (`io.quarkus:quarkus-resteasy-mutiny`) is present. If not, activate the extension by executing the following command:
+
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:add-extensions \
+    -Dextensions="io.quarkus:quarkus-resteasy-mutiny"
```

Comment:
The `io.quarkus:` shouldn't be necessary, but that works

Reply:
Good to know! Thanks.